src/main/resources/application.yaml

```diff
@@ -0,0 +1,56 @@
+heb:
+  togglr:
+    jwt:
+      secret:
+        id: 101
+      session:
+        seconds: 600
+      tokenheader: X-TOGGLR-TOKEN
+    oauth:
+      tokenheader: OAUTH-TOKEN
+      userIdentification: ${SSOUSERIDFIELD}
+      oauthEnabled: ${HEBTOGGLROAUTHENABLED}
+logging:
+  level:
+    com:
+      heb: debug
+    default: debug
+  path: logs
+server:
+  error:
+    whitelabel:
+      enabled: false
+  port: 8080
+  servlet:
+    context-path: /togglr-api
+  use-forward-headers: true
+spring:
+  application:
+    name: togglr_api
+  freemarker:
+    enabled: false
+  groovy:
+    template:
+      enabled: false
+  jmx:
+    enabled: false
+  main:
+    banner-mode: 'off'
+  thymeleaf:
+    cache: false
+  security:
+    oauth2:
+      client:
+        clientId: ${SSOCLIENTID}
+        clientSecret: ${SSOCLIENTSECRET}
+        accessTokenUri: ${SSOACCESSTOKENURI}
+        redirectUri: ${SSOREDIRECTURL}
+        userAuthorizationUri: ${SSOUSERAUTHORIZATIONURI} # query params client_id
+        clientAuthenticationScheme: form
+      resource:
+        userInfoUri: ${SSOUSERINFOURI}
+        preferTokenInfo: false
+app:
```

Comment:
Do we need this block?
I think we can remove this. The resource that David shared for SSO didn't have this. I got it from one of the older tutorial.

https://github.com/spring-projects/spring-security-oauth2-boot/blob/master/samples/spring-boot-sample-web-secure-github/src/main/resources/application.yml

Reply:
Oh you added this when starting the SSO work? Yeah we should be able to remove it then. H-E-B has their own jwt id and secret at the top. I'll make a note to take it out.